### PR TITLE
Fixes #575 : Language added to the API call .

### LINKED ISSUE #560: New Application Icon
Please exchange the application Icon, image is attached

susi_icon_2017

### LINKED ISSUE #575: Send ISO 639-1 of configured language to the server for queries
**Actual Behaviour**

When calling the `/susi/chat.json` api of susi_server, no language information is transmitted

**Expected Behaviour**

The configured language (not the country code) shall be send to the server.

Please add a query attribute `language` and fill it with the currently configured language using the ISO 639-1 standard (a lowercase two-letter code), see https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes

### PR DESCRIPTION
**Pull request by mayank408**
Fixes issue #575

Changes: Language object added in the API call.

### CODE REVIEW

**Comment by Abhi2424shek** [comment]
I think you got the error number incorrect

File: app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java [hunk 1]
```diff
@@ -1533,6 +1533,22 @@ protected void onResume() {
     }
 
 
+    public String getDefaultLocale(){
+
+        String language;
+        Locale defaultLocale = Locale.getDefault();
+        if(defaultLocale.toString().length()>2)
+            language = defaultLocale.toString().substring(0, Math.min(defaultLocale.toString().length(), 2));
```
Comment (the-dagger) [inline app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java]: No need to hack your way around this.
Android already allows you to get language code in different formats.

http://stackoverflow.com/a/23168383/5471095

**Comment by mayank408** [comment]
@the-dagger Changes done.

File: app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java [hunk 2]
```diff
@@ -1533,6 +1533,16 @@ protected void onResume() {
     }
 
 
+    public String getDefaultLocale(){
+
+        String language = Locale.getDefault().getLanguage();
```
Comment (the-dagger) [inline app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java]: We don't need a separate method for this imo
Simply use `Locale.getDefault().getLanguage()` in the api call.

**Comment by mayank408** [comment]
@the-dagger Changes done.

**Comment by mayank408** [comment]
@the-dagger @jig08  Requested changes done. Kindly merge this.

**Pull request merged**